source/client/player/input/Mouse.cpp

```diff
@@ -19,7 +20,7 @@ void Mouse::feed(ButtonType buttonType, ButtonState buttonState, int posX, int p
 	if (buttonType != NONE)
 		_inputs.push_back(MouseAction(buttonType, buttonState, posX, posY));
 
-	// Make sure button type is valid
+	// Make sure button texport __NV_PRIME_RENDER_OFFLOAD=1; export __GLX_VENDOR_LIBRARY_NAME=nvidiaype is valid
```

Comment:
i think you pasted a piece of code into this comment without realising

Reply:
I swear, middle-click paste on touchpads is the stupidest feature ever designed.